--- openlldp-1.1.1/Makefile.am.orig	2024-03-05 19:46:14.000000000 +0100
+++ openlldp-1.1.1/Makefile.am	2025-08-12 06:26:23.844817423 +0200
@@ -138,7 +138,7 @@ install-data-hook:	installdirs-local
 pkgconfigdir = ${libdir}/pkgconfig
 pkgconfig_DATA = lldpad.pc liblldp_clif.pc
 
-systemdsystemunitdir = $(prefix)/lib/systemd/system
+systemdsystemunitdir = /lib/systemd/system
 dist_systemdsystemunit_DATA = lldpad.service lldpad.socket
 
 bashcompletiondir = $(sysconfdir)/bash_completion.d
